lib/code_sponsor_web/templates/sponsorship/index.html.eex

```diff
@@ -32,6 +33,11 @@
                 <tr>
                   <td><%= link sponsorship.campaign.name, to: campaign_path(@conn, :show, sponsorship.campaign) %></td>
                   <td><%= link sponsorship.property.name, to: property_path(@conn, :show, sponsorship.property) %></td>
+                  <%= if sponsorship.creative do %>
```

Comment:
A minor thing but you could perhaps one line this since you'll have a `<td>` regardless of outcome.